src/js/utils/DSLUpdateUtil.js

```diff
@@ -118,10 +119,11 @@ const DSLUpdateUtil = {
       }
     }
 
-    // Bleed left or right whitespace (but not both)
+    // Bleed left whitespace, if exists, or right whitespace
+    // if we are the first token
```

Comment:
Unfortunately this function removes a token abruptly from the middle of the expression, so these left/right bleeds are used as a best-effort clean-up of the expression. However there should be a cleanup post-processing phase, that I will add later.

(Fixed the comment to be more clear)